README.md

```diff
@@ -93,6 +96,8 @@
 
 **注意：您在此伺服器中意味著接受這些規則，包括所有進一步的更改。這些更改可能隨時進行，恕不另行通知，您有義務定期自行查看本規則。**  
 
+**以上規則奶茶伺服器保有最終解釋權。**
```

Comment:
Is this really need?

Reply:
Rules gotta be clearer.
To prevent some unnecessary conflicts .w.